dev-docs/multipart-subscriptions-protocol.md

```diff
@@ -4,13 +4,12 @@ Instead of relying on WebSockets, the subscriptions protocol supported by the ro
 
 ## Communication
 
-When sending a request containing a subscription to the router, clients should include the following `Accept` header to indicate their support for the multipart subscriptions protocol:
+When sending a request containing a subscription to the router, clients MUST support the `multipart/mixed; subscriptionSpec="1.0"` content-type in addition to the `application/graphql-response+json` content-type:
+
 ```
-Accept: multipart/mixed; boundary="graphql"; subscriptionSpec="1.0", application/json
+Accept: multipart/mixed; subscriptionSpec="1.0", application/graphql-response+json
```

Comment:
I also changed the non-subscription mime type to `application/graphql-response+json` as recommended by [GraphQL over HTTP](https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md). Happy to revert that change and make it part of another discussion.

Reply:
BTW we have to still be backward compatible with what's existing right now to not break existing implementations

Reply:
> why not both 

I'd this spec should be as strict as possible to avoid confusion. If the router supports `application/graphql-response+json` then use that one. If not, let's stick with `application/json` and wait before changing it.

Clients can always decide to send more if they want.

Reply:
`application/graphql-response+json` isn't official yet, so let's hold off.

Reply:
In any cases I think we won't be able to be that strict if we don't want to introduce breaking changes

Reply:
> let's hold off.

Sone in https://github.com/apollographql/router/pull/4395/commits/8cdd3b081830bf421a4d9fe4d3a7dfef2cd10e27